gnovm/pkg/gnolang/op_expressions.go

```diff
@@ -194,8 +194,12 @@ func (m *Machine) doOpRef() {
 			nv.Value = rv2
 		}
 	}
+	elt := xv.TV.T
+	if elt == DataByteType {
```

Comment:
Can you put a comment here explaining why this is an exception?

Reply:
added.